src/table/body-cell/styles.scss

```diff
@@ -26,11 +26,33 @@ $edit-button-padding-right: calc(
 ); // Cell vertical padding + xxs space that would normally come from the button.
 $success-icon-padding-right: calc(#{$edit-button-padding-right} + #{$icon-width-with-spacing});
 
+.expandable-toggle-wrapper {
+  position: absolute;
+  inset-block: 0;
+  display: flex;
+  align-items: center;
+  margin-inline-start: calc(-1 * (#{awsui.$space-m} + #{awsui.$space-xs}));
+}
+
+@mixin cell-offset($padding) {
+  padding-inline-start: $padding;
+
+  @for $i from 1 through 9 {
```

Comment:
Do we have a screenshot test for this "extreme nesting" case of 10+?

Reply:
Yes, the extreme nesting case is the first permutation.

Reply:
Looks like there is no indication at all of further nesting past the 9th level. Would it be possible to hard-disallow more than 9 levels instead?

Reply:
It is possible, but we decided not to. Sometimes, it is the end user who decides the shape of the data and we don't want that data to become inaccessible. The recommendation though will be to avoid having deep nesting and add mechanisms to prevent deep nesting on the consumer end.